heron/packing/tests/java/com/twitter/heron/packing/binpacking/FirstFitDecreasingPackingTest.java

```diff
@@ -197,36 +229,36 @@ public void testContainerRequestedResources() throws Exception {
 
 
     Assert.assertEquals(Math.round(7 * instanceCpuDefault
-            + (10 * 7 * instanceCpuDefault) / 100 + instanceCpuDefault),
+            + (DEFAULT_CONTAINER_PADDING * 7 * instanceCpuDefault) / 100 + instanceCpuDefault),
```

Comment:
Can we replace 7 with `totalInstances`? The more we clearly define the integers users, the easier it is to understand.

```int totalInstances = spoutParallelism + boltParallelism```